lib/backend/k8s/conversion.go

```diff
@@ -83,10 +83,10 @@ func (c Converter) parseProfileName(profileName string) (string, error) {
 	return strings.TrimPrefix(profileName, "k8s_ns."), nil
 }
 
-// NamespaceToProfile converts a Namespace to a Calico Profile.  The Profile stores
+// namespaceToProfile converts a Namespace to a Calico Profile.  The Profile stores
```

Comment:
I think it needs to be exported for the Golang policy controller

Reply:
hmm.  i'll try and rebase - this shouldn't have changed.

Reply:
grrr - rebase didn't fix.  ok  i'll fix up by hand.